src/main/java/racinggame/domain/RacingGame.java

```diff
@@ -0,0 +1,54 @@
+package racinggame.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGame {
+    private static List<Car> cars = new ArrayList<>();
+
+    public RacingGame(String names) {
+        this(createCars(names));
+
+    }
+
+    public RacingGame(List<Car> cars) {
+        RacingGame.cars = cars;
+    }
+
+    private static List<Car> createCars(String names) {
+        String[] split = names.split(",");
+
+        for (String s : split) {
+            cars.add(new Car(s.trim()));
+        }
```

Comment:
```suggestion
        List<Car> cars = new ArrayList<>()
        for (String s : split) {
            cars.add(new Car(s.trim()));
        }
```
여기서 ArrayList를 직접 생성해 구현하면 된다.